src/testEnvironments.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Do we need 2 copies of this file ? (there is one copy in the root folder and one in the src folder)

Reply:
Fixed that. I originally tried it in the src folder, but the constraints of the remote test execution are that the built binaries must be in a subfolder of the folder that the solution file lives in. This is why I moved the .sln files and the testEnvironments.json file to the root of the repo.